lib/src/auth/cryptosign.dart

```diff
@@ -7,7 +7,7 @@ import "package:wampproto/src/auth/auth.dart";
 const hex = Base16Encoder.instance;
 
 class CryptoSignAuthenticator extends IClientAuthenticator {
-  CryptoSignAuthenticator(String authID, this._privateKeyHex, [Map<String, dynamic>? authExtra]) : super(type, authID) {
+  CryptoSignAuthenticator(this._privateKeyHex, String authID, [Map<String, dynamic>? authExtra]) : super(type, authID) {
```

Comment:
This order is wrong, use the order we have in Go impl. authid, extra, private_key